libr/include/r_bin.h

```diff
@@ -14,6 +14,10 @@ extern "C" {
 
 R_LIB_VERSION_HEADER (r_bin);
 
+#define MODE_PRINT 0x000
```

Comment:
this is used in core mainly, but its a pretty generic "types", so i would kill it or make it into r_util and reuse it everywhere, but basically we handle 'j' for json, 'r' for r2 (or '*', 0 for normal, 'q' for quiet (simple)

Reply:
Yeah probably moving to r_util is the best thing... I see they are dupped in r_core.h (e.g. `R_CORE_BIN_PRINT`, etc.)

Reply:
Yeah probably moving to r_util is the best thing... I see they are dupped in r_core.h (e.g. `R_CORE_BIN_PRINT`, etc.)

Reply:
there's echo? pls move this and the ones from core.. i would probably put that in r_types or something even if its not a basic enum, it is for the whole r2, and all the printing modes should be implemented in core. but we are far from this point too :P